### PR TITLE
[TACKLE-9] Add 'Assessment' column to the Application list page

### PR DESCRIPTION
**Pull request by carlosthe19916**
Resolves https://github.com/konveyor/tackle-ui/issues/15

### CODE REVIEW

**Comment by carlosthe19916** [comment]
@mrizzi just to let you know that this PR is ready to be reviewed

**Review by mrizzi — COMMENTED** [review]
depends on https://github.com/konveyor/tackle-controls/pull/76

File: src/shared/components/status-icon-assessment/status-icon-assessment.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  [key in StatusIconAssessmentType]: {
+    Icon:
+      | React.ComponentClass<SVGIconProps>
+      | React.FunctionComponent<SpinnerProps>;
+    color: { name: string; value: string; var: string };
+  };
+};
+const iconList: IconListType = {
+  NotStarted: {
+    Icon: ErrorCircleOIcon,
```
Comment (mrizzi) [inline src/shared/components/status-icon-assessment/status-icon-assessment.tsx:33]: From mockups it seems like the icon should be dark with a white `x` 
Screenshot from 2021-04-07 13-08-45
but it seems like the opposite has been used
Screenshot from 2021-04-07 13-09-33

Maybe `TimesCircleIcon` icons can be the right one?

**Review by mrizzi — COMMENTED** [review]
Minor issue with the icon for `Not started`

File: src/shared/components/status-icon-assessment/status-icon-assessment.tsx (hunk 1 above)
Comment (carlosthe19916) [reply src/shared/components/status-icon-assessment/status-icon-assessment.tsx]: Good point, my last commit changed the icon. Thanks @mrizzi

**Review by mrizzi — APPROVED** [review]
Tested with Pathfinder backend only for `Not started` value.
Tested `Completed` and `In-progress`:
* icons using `yarn storybook`
* labels checking manually the terms file

Pathfinder remote call when loading the application list page verified to be `/api/pathfinder/assessments?applicationId=1`

Thanks @carlosthe19916 :+1:

**Pull request merged**